test/test_data/international_components_for_unicode.py

```diff
@@ -4,30 +4,50 @@
 mapping_test_data = [
     {
         "product": "international_components_for_unicode",
-        "version": "3.8.1",
-        "version_strings": ["CLIENT icu-release-3-8-1"],
+        "version": "60.3",
+        "version_strings": ["ICU 60.3"],
     },
     {
         "product": "international_components_for_unicode",
-        "version": "3.8.1",
-        "version_strings": ["ICU 3.8.1"],
+        "version": "60.3",
+        "version_strings": ["icu-60.3"],
     },
+]
+package_test_data = [
     {
+        "url": "http://www.rpmfind.net/linux/openmandriva/cooker/repository/x86_64/main/release/",
+        "package_name": "icu-68.2-1-omv4002.x86_64.rpm",
         "product": "international_components_for_unicode",
-        "version": "3.8.1",
-        "version_strings": ["icu-3.8.1"],
+        "version": "68.2",
     },
     {
+        "url": "http://mirror.centos.org/centos/8/BaseOS/aarch64/os/Packages/",
+        "package_name": "icu-60.3-2.el8_1.aarch64.rpm",
         "product": "international_components_for_unicode",
-        "version": "3.8.1",
-        "version_strings": ["icu_3.8.1"],
+        "version": "60.3",
     },
-]
-package_test_data = [
     {
-        "url": "http://www.rpmfind.net/linux/openmandriva/cooker/repository/x86_64/main/release/",
-        "package_name": "icu-68.2-1-omv4002.x86_64.rpm",
+        "url": "http://archive.ubuntu.com/ubuntu/pool/main/i/icu/",
+        "package_name": "icu-devtools_67.1-4_amd64.deb",
         "product": "international_components_for_unicode",
-        "version": "68",
-    }
-]
+        "version": "67.1",
+    },
+    {
+        "url": "https://ftp.netbsd.org/pub/pkgsrc/packages/NetBSD/amd64/8.2/All/",
+        "package_name": "icu-68.2.tgz",
+        "product": "international_components_for_unicode",
+        "version": "68.2",
+    },
+    {
+        "url": "https://rpmfind.net/linux/dag/redhat/el4/en/x86_64/dag/RPMS/",
+        "package_name": "icu-3.6-1.el4.rf.x86_64.rpm",
+        "product": "international_components_for_unicode",
+        "version": "3.6",
+    },
+    # {
+    #     "url": "https://rpmfind.net/linux/dag/redhat/el4/en/x86_64/dag/RPMS/",
+    #     "package_name": "icu-2.6.2-1.2.el4.rf.x86_64.rpm",
+    #     "product": "international_components_for_unicode",
+    #     "version": "2.6.2",
+    # },
```

Comment:
`icu-2.6.2-1.2.el4.rf.x86_64.rpm` -> this is commented out because, it had no good version patterns (It only contained one "2.6.2" which is not recommended). This was the oldest package that I was able to download.
`icu-3.6-1.el4.rf.x86_64.rpm` -> from this version, the version patterns are consistent. 

Also, I was not able to get any icu-v.v.v package (of type icu-3.8.1). If anyone finds one, please link them down below, I'll test them.